File: DataSummitSolution/DataSummitUI/src/app/efModels/country.ts
import { Address } from "../companies/models/address";

export class Country {

	CountryId: number;
	Iso: string;
	Name: string;
	SentenceCaseName: string;
	Iso3: string;
	Numcode?: number;
	Phonecode: number;
	CreatedDate?: Date | string;
	UserId?: number;
	Addresses: Address[];

	constructor(countryId?: number, iso?: string, name?: string,
		sentenceCaseName?: string, iso3?: string, numcode?: number,
		phonecode?: number, createdDate?: Date | string, userId?: number,
		addresses?: Address[])
	{
		this.CountryId = countryId || 0;
		this.Iso = iso || "";
		this.Name = name || "";
		this.SentenceCaseName = sentenceCaseName || "";
		this.Iso3 = iso3 || "";
		this.Numcode = numcode || 0 || null;
		this.Phonecode = phonecode || 0;
		this.CreatedDate = createdDate || new Date(Date.now()) || null;
		this.UserId = userId || 0 || null;
		this.Addresses = addresses;
	}
}

File: DataSummitSolution/DataSummitModels/DB/Country.cs
using System;
using System.Collections.Generic;

#nullable disable

namespace DataSummitModels.DB
{
    public partial class Country
    {
        public Country()
        {
            Addresses = new HashSet<Address>();
        }

        public short CountryId { get; set; }
        public string Iso { get; set; }
        public string Name { get; set; }
        public string SentenceCaseName { get; set; }
        public string Iso3 { get; set; }
        public short? Numcode { get; set; }
        public int Phonecode { get; set; }
        public DateTime? CreatedDate { get; set; }
        public long? UserId { get; set; }

        public virtual ICollection<Address> Addresses { get; set; }
    }
}
